classes/test.php

```diff
@@ -1181,6 +1181,12 @@ public function runTestMethod($testMethod, array $tags = array())
 						)
 					;
 
+					$this->factoryBuilder->build($testedClass)
+						->addToAssertionManager($this->assertionManager, 'callStaticOnTestedInstance', function($methodName) use ($testedClass) {
```

Comment:
you don't need to use the factory builder in your case ;)

You should rather use something like : https://github.com/atoum/atoum/blob/master/classes/test.php#L371

```php
$this->assertionManager
    ->setHandler('callStaticOnTestedInstance', function () use ($test) {
            $args = func_get_args();
            $method = array_shift($args);

            return call_user_func_array(array($test->getTestedCLassName(), $method), $args);
        }
    )
; 
```